cookbook/integrations/flytekit_plugins/modin_examples/applymap_function.py

```diff
@@ -0,0 +1,80 @@
+"""
+``applymap`` Function
+----------------------------
+
+In this example, let's manipulate the Modin DataFrame using the ``applymap`` function.
+
+"""
+
+# %%
+# Import ``ray`` and close the previous instances (if any) and start a new instance.
+import ray
+
+ray.shutdown()  # close previous instance of ray
+ray.init()  # open a new instance of ray
+
+import time
+import typing
+from types import ModuleType
+from typing import Tuple
+
+import flytekitplugins.modin
+
+# %%
+# Next, import the necessary dependencies.
```

Comment:
This line should be present above `import time`. Also, have you run isort? I don't think `import ray` has to be present there. Please move it somewhere between the other import statements and run isort again. Also, you can initialize ray after importing the libraries.